src/System.ComponentModel.Composition/src/Strings.resx

```diff
@@ -0,0 +1,153 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should not check in any strings.designer.cs files

Generally there should be exactly one strings.resx file and it should be in src/Resources, not here. Please check...

Reply:
Also neither strings.resx nor strings.designer.cs should be in the project file at all. strings.resx is automatically found in the resources folder